### PR TITLE
MS17291: Create-app increase focus distance limit

### PR DESCRIPTION
**Pull request by thoys**
Fixes https://highfidelity.manuscript.com/f/cases/17291/You-can-t-move-the-camera-out-all-the-way

## Testplan
1. Open the create app in desktop mode
2. Create a Box of size 1000x1000x1000
3. Use the `F` key to focus on the entity
4. Zoom all the way out using the scroll wheel.
5. The Edit handles should still be visible and move-able

### CODE REVIEW

File: scripts/system/libraries/entityCameraTool.js [hunk 1]
```diff
@@ -255,14 +255,6 @@ CameraManager = function() {
         that.updateCamera();
     }
 
-    that.getZoomPercentage = function() {
-        return (that.zoomDistance - MIN_ZOOM_DISTANCE) / MAX_ZOOM_DISTANCE;
-    }
-
-    that.setZoomPercentage = function(pct) {
-        that.targetZoomDistance = pct * (MAX_ZOOM_DISTANCE - MIN_ZOOM_DISTANCE);
-    }
```
Comment (thoys) [inline scripts/system/libraries/entityCameraTool.js]: These functions were unused, were previously used for a zoom bar which is kind of useless with the new max zoom distance.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12020/HighFidelity-Beta-PR13804-925cfc9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12020/HighFidelity-Beta-Interface-PR13804-925cfc9.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12020/HighFidelity-Beta-PR13804-925cfc9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12020/HighFidelity-Beta-Interface-PR13804-925cfc9.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12063/HighFidelity-Beta-PR13804-f26188d.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12063/HighFidelity-Beta-PR13804-f26188d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12063/HighFidelity-Beta-Interface-PR13804-f26188d.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12063/HighFidelity-Beta-PR13804-f26188d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12063/HighFidelity-Beta-Interface-PR13804-f26188d.dmg

**Comment by QliCknox** [comment]
Currently Testing

**Comment by QliCknox** [comment]
PASS @conklin94122 

Edit handles still appear while the user is focused and scrolled all the way out

**Pull request merged**